project3/index.html

```diff
@@ -0,0 +1,409 @@
+                                style="stroke-width: 0.564843px; fill: rgb(255, 255, 255); stroke: rgb(255, 255, 255);" />
+                        </g>
+                        <g style="" transform="matrix(0.883978, 0, 0, 0.886427, 0.396386, -16.505608)">
+                            <path
+                                d="M 61.767 27.286 C 61.867 27.386 62.067 27.486 62.267 27.486 C 62.467 27.486 62.667 27.386 62.767 27.286 C 63.067 26.986 63.067 26.486 62.767 26.286 L 60.967 24.486 C 60.667 24.186 60.167 24.186 59.967 24.486 C 59.667 24.786 59.667 25.286 59.967 25.486 L 61.767 27.286 Z"
+                                style="stroke-width: 0.564843px; fill: rgb(255, 255, 255); stroke: rgb(255, 255, 255);" />
+                            <path
+                                d="M 67.467 33.086 C 67.567 33.186 67.767 33.286 67.967 33.286 C 68.167 33.286 68.367 33.186 68.467 33.086 C 68.767 32.786 68.767 32.286 68.467 32.086 L 66.667 30.286 C 66.367 29.986 65.867 29.986 65.667 30.286 C 65.467 30.586 65.367 31.086 65.667 31.286 L 67.467 33.086 Z"
+                                style="stroke-width: 0.564843px; fill: rgb(255, 255, 255); stroke: rgb(255, 255, 255);" />
+                            <path
+                                d="M 60.467 33.286 C 60.667 33.286 60.867 33.186 60.967 33.086 L 62.767 31.286 C 63.067 30.986 63.067 30.486 62.767 30.286 C 62.467 29.986 61.967 29.986 61.767 30.286 L 59.967 31.986 C 59.667 32.286 59.667 32.786 59.967 32.986 C 60.067 33.186 60.267 33.286 60.467 33.286 Z"
+                                style="stroke-width: 0.564843px; fill: rgb(255, 255, 255); stroke: rgb(255, 255, 255);" />
+                            <path
+                                d="M 66.267 27.486 C 66.467 27.486 66.667 27.386 66.767 27.286 L 68.567 25.486 C 68.867 25.186 68.867 24.686 68.567 24.486 C 68.267 24.286 67.767 24.186 67.567 24.486 L 65.767 26.286 C 65.467 26.586 65.467 27.086 65.767 27.286 C 65.867 27.386 66.067 27.486 66.267 27.486 Z"
+                                style="stroke-width: 0.564843px; fill: rgb(255, 255, 255); stroke: rgb(255, 255, 255);" />
+                        </g>
+                        <g transform="matrix(1, 0, 0, 1, -3.770447, 0.000099)">
+                            <circle cx="8.439" cy="23.002" r="1.015" style="fill: rgb(255, 255, 255);" />
+                            <circle cx="8.439" cy="3.724" r="1.015" style="fill: rgb(255, 255, 255);" />
+                            <path
+                                d="M 9.453 7.782 C 9.453 7.222 8.999 6.768 8.439 6.768 C 7.879 6.768 7.424 7.222 7.424 7.782 L 7.424 18.943 C 7.424 19.504 7.879 19.958 8.439 19.958 C 8.999 19.958 9.453 19.504 9.453 18.943 L 9.453 7.782 Z"
+                                style="fill: rgb(255, 255, 255);" />
+                        </g>
+                    </svg>
+                    <label class="strengh-label" for="strengh">Strengh</label>
+                    <select class="select-reset" name="" id="">
```

Comment:
This `input` is missing the `id` to match it with its `label`
```suggestion
                    <select class="select-reset" id="strengh">
```